charts/kamaji/Chart.yaml

```diff
@@ -15,7 +15,7 @@ name: kamaji
 sources:
 - https://github.com/clastix/kamaji
 type: application
-version: 0.12.9
+version: 0.12.10
```

Comment:
Let's bump the minor version, this can be considered as a feature.

Reply:
Ok ! Do I change the commit as "feat: add tolerations on etcd sts" so ?